src/test/kotlin/minesweeper/domain/LandTest.kt

```diff
@@ -0,0 +1,38 @@
+package minesweeper.domain
+
+import minesweeper.domain.tile.Tiles
+import minesweeper.domain.tile.pos.Coordinate
+import minesweeper.domain.tile.state.set.Mine
+import minesweeper.domain.tile.state.set.NotChecked
+import org.junit.jupiter.api.Test
+
+class LandTest {
+    @Test
+    fun `Land - 문자열 타일 목록 반환 테스트`() {
+        // given
+        val tiles = Tiles(
+            arrayListOf(
+                Mine(Coordinate.of(0, 0)), NotChecked(Coordinate.of(0, 1), false),
+                NotChecked(Coordinate.of(1, 0), false), Mine(Coordinate.of(1, 1))
+            )
```

Comment:
```suggestion
            listOf(
                Mine(Coordinate.of(0, 0)), NotChecked(Coordinate.of(0, 1), false),
                NotChecked(Coordinate.of(1, 0), false), Mine(Coordinate.of(1, 1))
            )
```
List 타입이므로 굳이 ArrayList를 만들지 않아도 될 것 같아요. 🙂

Reply:
네, 조언해 주신 내용 확인해 보니 굳이 `ArrayList` 사용은 불필요해 보이네요 🥲 제안해 주신 부분 그대로 적용하였습니다! 감사합니다 🙏😄

- https://github.com/next-step/kotlin-minesweeper/commit/4376db1111a01e136635e8a9aa5695da7b8da7da